ming_sim/due_review.py

```
@@ -0,0 +1,423 @@
+        "noop": False,
+    }
+
+
+def apply_due_review_for_todo(
+    db: Any,
+    state: Any,
+    todo: Dict[str, object],
+    *,
+    commit: bool = False,
+) -> Dict[str, object]:
+    """单条正式复核：落格（有案卷）或只场面消费（无案卷）+ todo 消费。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)